src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -100,9 +100,41 @@ public partial class FluentDataGrid<TGridItem> : FluentComponentBase, IHandleEve
     [Parameter]
     public DataGridResizeType? ResizeType { get; set; }
 
+    /// <summary>
+    /// Label used for the text field in the resize options dialog.
+    /// Defaults to "Column width (in pixels)".
+    /// </summary>
     [Parameter]
     public string ResizeLabel { get; set; } = "Column width (in pixels)";
 
+    /// <summary>
```

Comment:
To avoid these multiple parameters, and group all this information in a single object, how about using something like this :

```csharp
public record ResizeAria
{
    public string? GrowLabel { get; set; }
    public string? ShrinkLabel { get; set; }
    public string? ResetLabel { get; set; }
    public string? SubmitLabel { get; set; }

    public static ResizeAria Default => new ResizeAria();
}
```

The usage will be:

```xml
<FluentDataGrid ResizeAria="@(ResizeAria.Default with { GrowLabel = "...", ShrinkLabel = "..." } )">
```

Do you find it too complex ?

Reply:
I was thinking about something like that as well (but then with a dictionary of some sorts)

Reply:
A record has been added. See updates in PR message above

Reply:
1. I find the syntax for defining the record clearer using property declarations (as above). And using your syntax, we don't have XML comments / property documentation.

2. Why did you put this class in the `Infrastructure` folder and not in the `DataGrid/Columns` folder?

3. I think we need a static property `Default` to create an instance of this object. Thus, the developer can initialize the parameter using `ResizeAria.Default with { GrowLabel = “ ...”, ShrinkLabel = “ ...” }`

4. ⚠️ You can't force RPs to merge: without explicit approval, we can continue discussing.

Reply:
1) By doing it this way, the developer does not have to do anything to use the default values.  I think we can put the comments about the fields of the record in the parameter declaration in the Grid
2) Yes, that is a mistake. Will move them
3) ~~I could not get your variant to work with implemented defaults. I don't want to force the developer to **have to** supply values in the default case~~ I figured it out. Will change it in the other PR
4) I know but I wanted this in because (and was too impatient). I need it in the other grid thing I'm working on (PR inbound) (mea culpa). We can still discuss and change because we are the end bosses.